src/containers/Network/Validators.tsx

```diff
@@ -0,0 +1,96 @@
+import React, { useEffect, useState } from 'react'
+import axios from 'axios'
+import { useTranslation } from 'react-i18next'
+import NetworkTabs from './NetworkTabs'
+import Streams from '../shared/components/Streams'
+import ValidatorsTable from './ValidatorsTable'
+import Log from '../shared/log'
+import { localizeNumber } from '../shared/utils'
+import { useLanguage } from '../shared/hooks'
+import Hexagons from './Hexagons'
+
+export const Validators = () => {
+  const language = useLanguage()
+  const { t } = useTranslation()
+  const [vList, setVList] = useState<any>([{}])
+  const [validations, setValidations] = useState([])
+  const [metrics, setMetrics] = useState({})
+  const [unlCount, setUnlCount] = useState(0)
+
+  useEffect(() => {
+    fetchData()
+    const interval = setInterval(fetchData, 5000)
```

Comment:
This interval is the same as before - is it worth factoring out into a single constant, or should it legitimately be configured separately?

Reply:
Oh nevermind - took me a sec to realize this is the same file just ts version - oops!